### PR TITLE
Prepare for v1.6.0

### PR DESCRIPTION
**Pull request by ianvernon**
Signed-off by: Ian Vernon <ian@cilium.io>

---
This change is [Reviewable](https://reviewable.io/reviews/cilium/cilium/8940)

### CODE REVIEW

File: install/kubernetes/quick-install.yaml [hunk 1]
```diff
@@ -382,7 +382,7 @@ spec:
               name: cilium-config
               optional: true
         image: "docker.io/cilium/cilium:latest"
```
Comment (joestringer) [inline install/kubernetes/quick-install.yaml]: These image tags should be updated too, no?

**Review by joestringer — CHANGES_REQUESTED** [review]

File: install/kubernetes/quick-install.yaml (hunk 1 above)
Comment (ianvernon) [reply install/kubernetes/quick-install.yaml]: Fixed!

File: install/kubernetes/quick-install.yaml [hunk 2]
```diff
@@ -469,7 +469,7 @@ spec:
...
```
Comment (tgraf) [inline install/kubernetes/quick-install.yaml]: This may be a bug in the helm chart. Still pointing to latest

**Review by tgraf — CHANGES_REQUESTED** [review]

File: install/kubernetes/quick-install.yaml (hunk 2 above)
Comment (ianvernon) [reply install/kubernetes/quick-install.yaml]: Fixed manually:

```
$ grep "latest" quick-install.yaml
$
```

Can file an issue to make sure the `quick-install.yaml` chart is updated when we run `make` as well so as to not block the release.

File: install/kubernetes/quick-install.yaml [hunk 3]
```diff
@@ -482,6 +482,7 @@ spec:
         - mountPath: /var/run/cilium
           name: cilium-run
       restartPolicy: Always
+      priorityClassName: system-node-critical
```
Comment (tgraf) [inline install/kubernetes/quick-install.yaml]: This needs to be removed in order to be compatible with all versions of k8s. I'll add a PR to run a `sed` to remove it so we don't depend on the k8s version context `helm` is in when a developer runs it.

**Review by tgraf — CHANGES_REQUESTED** [review]

**Review by tgraf — APPROVED** [review]

**Review by joestringer — APPROVED** [review]

**Pull request merged**